### ISSUE TITLE
Changes to the TripalField class

### ISSUE THREAD

**Issue by spficklin** [statonlab/fields_generator#31]
I have just made a change to the TripalField class (on the **collections** branch of Tripal) which moves the download_formatters out of the $default_instance_settings variable and into it's own static variable.  This is because it would be really hard to add new formatters to a field without manually rewritting the settings for existing fields.... too much effort.   Once the 'collections' branch gets merged into the 7.x-3.x branch then this change will be active.

**Comment by almasaeed2010** [comment]
Thanks for pointing this out @spficklin 

We added a fix in #32 that we'll merge once the collections branch is merged.

**Comment by bradfordcondon** [comment]
I wonder if there is some way to set up a github alert when the base class files change so we know to update our stubs.

**Comment by bradfordcondon** [comment]
Thanks Stephen for the heads up.  now that it's been merged I'm merging the PR and closing the issue.
